iocore/net/UnixNetAccept.cc

```diff
@@ -162,10 +162,8 @@ NetAccept::init_accept(EThread *t)
 }
 
 void
-NetAccept::init_accept_per_thread()
+NetAccept::init_accept_per_thread(EThread *thread)
```

Comment:
Change this to `initialize_thread_for_accept` to be thematically consistent with `initialize_thread_for_net`.